### DISCUSSION TITLE
Bot read people's custom status and if it says some specific sentence apply a role!

### DISCUSSION THREAD

**Discussion by yigitrgeles** [discordjs/discord.js#6796] [Q&A] [answered]
How can i do that?

**Comment by almostSouji** [comment, marked as answer]
Custom status is an activity and part of  [`GuildMember#presence`](https://discord.js.org/#/docs/main/stable/class/GuildMember) > [`Presence#activities`](https://discord.js.org/#/docs/main/stable/class/Presence?scrollTo=activities) with the [`.type`](https://discord.js.org/#/docs/main/stable/class/Activity?scrollTo=type) `CUSTOM`.

A message author's custom status could be accessed as:
```js
const cs = message.member.presence.activities.find(activity => activity.type === "CUSTOM")
```

If you want to listen to updates in guild member's presences, you can do so via the [`Client#presenceUpdate`](https://discord.js.org/#/docs/main/stable/class/Client?scrollTo=e-presenceUpdate) event. Note that presences are locked behind a [privileged intent](https://discordjs.guide/popular-topics/intents.html#privileged-intents).
